docs/reportData.html

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: Reports Data API
+---
+
+<p>Campaign provides an API to fetch reports data. Just like all APIs in the SDK, it's been wrapped into a function and will return a XML or JSON object based on the requested representation</p>
+
+<p>Function Definition</p>
+<pre class="code">
+client.getReport = (callContext: any, representation: string) : Promise<any>
+</pre>
+<p>callContext</p>
+<pre class="code">
+{
+  reportName: string<report Name>,
+  context: <"selection"|"global">,
+  selection: string<Delivery Id>,
+  selectionCount: number<count of deliveries>,
```

Comment:
Same as above. It's also not clear why you need this in addition to the selection. It's probably a issue with the API itself, but I'd like to use the SDK as an opportunity to clarify and properly document those issues. Can you add some details as what this value should be and how users of the SDK can set it. Can it be checked by the SDK?